### DISCUSSION TITLE
[BS4] Showing 2 icons in select-menu

### DISCUSSION THREAD

**Discussion by marceloverdijk** [twbs/bootstrap#31140] [General] [answered]
I'm testing a [select-menu](https://getbootstrap.com/docs/4.5/components/forms/#select-menu) like:

```
<select class="custom-select">
  <option selected>Open this select menu</option>
  <option value="1">One</option>
  <option value="2">Two</option>
  <option value="3">Three</option>
</select>
```

but it shows 2 icons in my project:

![image](https://user-images.githubusercontent.com/197004/85234448-b1df5700-b40d-11ea-9f84-38c211e2ef08.png)

The example [here](https://getbootstrap.com/docs/4.5/components/forms/#select-menu) only shows only 1 icon:

![image](https://user-images.githubusercontent.com/197004/85234462-d63b3380-b40d-11ea-9dc7-72425eb1a3bb.png)

The difference is that the BS example has:

```
.custom-select {
    ..
    -webkit-appearance: none;
    ..
}
```

![image](https://user-images.githubusercontent.com/197004/85234531-3af68e00-b40e-11ea-807e-f2260d2899b9.png)

and my project does not seem to have that one. When I add it I only get 1 icon as I would expect out of the box.

Diving deeper in the [custom-forms.scss](https://github.com/twbs/bootstrap/blob/v4.5.0/scss/_custom-forms.scss#L219-L234) source code I see no mention of `-webkit-appearance: none;` so I wonder how it works it in the example?

**Comment by Lausselloic** [comment, marked as answer]
Hello, yes it's the postcss task that adds the browser prefix in output css files https://github.com/twbs/bootstrap/blob/v4-dev/package.json#L38 so you won't find prefix in scss file, and you need to add a postcss task in your project if you recompile scss files from source

**Reply by marceloverdijk** [reply]
Thx @Lausselloic I indeed didn't have the postcss setup, I will look into it.
